### PR TITLE
Compare obligations

### PR DESCRIPTION
**Pull request by rudra-superrr**
Feature which compares the previous version obligations with current version of the project and should be able to differentiate the newly added components in the present version.
Can differentiate by change in color of the component name in the table.

### CODE REVIEW

File: messages/de.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "Enter obligation comment": "Verpflichtungskommentar eingeben",
```
Comment (hoangnt2) [inline messages/de.json:2]: Move this into "default" (same with other files)

Comment (heliocastro) [reply messages/de.json:2]: @rudra-superrr What is the status of this one ?

**Comment by rudra-superrr** [comment]
Hi @heliocastro @hoangnt2, made the requested changes.

File: src/app/[locale]/projects/components/Obligations/ObligationsView/LicenseObligation.tsx [hunk 2]
```diff
@@ -0,0 +1,480 @@
...
+            } catch (e) {
+                console.error(e)
+            }
+        };
+
+        loadData();
+        return () => {
+            controller.abort();
+        };
+    }, [projectId, status, selectedProjectId, payload]);
```
[the remark this answers is not in the document]
Comment (heliocastro) [reply src/app/[locale]/projects/components/Obligations/ObligationsView/LicenseObligation.tsx]: Thank you

**Comment by heliocastro** [comment]
@rudra-superrr You need sign the commits, i can't merge it like that ( sorry )

**Pull request merged**